Resources/Prototypes/Entities/Mobs/Player/observer.yml

```diff
@@ -60,6 +57,11 @@
     baseSprintSpeed: 12
     baseWalkSpeed: 8
   - type: MovementIgnoreGravity
+  - type: Physics
+    bodyType: KinematicController
+    fixedRotation: true
```

Comment:
```suggestion
```

Reply:
observer had this previously, i just moved it